Script/AtomicEditor/hostExtensions/languageExtensions/TypscriptLanguageExtension.ts

```diff
@@ -337,6 +341,42 @@ export default class TypescriptLanguageExtension implements Editor.HostExtension
         WebView.WebBrowserHost.setGlobalStringProperty("TypeScriptLanguageExtension", "tsConfig", JSON.stringify(tsConfig));
     }
 
+    generateExternalProject ()
+    {
+      var projectPath : string = ToolCore.toolSystem.project.projectPath;
+
+      // Create TypeScript folder in project root
+      var tsPath : string = projectPath + "TypeScript/";
+      Atomic.getFileSystem().createDir(tsPath);
+
+      // Copy the Atomic.d.ts definition file to the TypeScript folder
+      var toolDataDir : string = ToolCore.toolEnvironment.toolDataDir;
+      Atomic.getFileSystem().copy(toolDataDir + "/TypeScriptSupport/Atomic.d.ts", tsPath + "Atomic.d.ts");
+
+      // Generate a tsconfig.json file
+      var config = new Atomic.File(projectPath + "tsconfig.json", Atomic.FILE_WRITE);
+      config.writeString
+(`
+{
+    "compilerOptions": {
```

Comment:
At the top of this module is an object called ```defaultCompilerOptions``` which is what is passed to the Monaco editor.  You should be able to use that and perform a ```JSON.stringify``` to write it to the file.